packages/admin/src/tabs/RouterTabs.tsx

```diff
@@ -33,12 +34,15 @@ function RouterTabsComponent({
     classes,
 }: Props & WithStyles<typeof styles>) {
     const childrenArr = React.Children.toArray(children);
+    const finalFormContext = useFinalFormContext();
 
     const handleChange = (event: {}, value: number) => {
         const paths = childrenArr.map((child) => {
             return React.isValidElement<TabProps>(child) ? child.props.path : null;
         });
-        history.push(match.url + paths[value]);
+        history.push(match.url + paths[value], {
+            parentFormId: finalFormContext.formId,
```

Comment:
This state is not there when eg. copy/pasting this url in a new browser window. What happens then?

Reply:
I don't understand the problem. The state is completely optional, if it's not set nothing happens.